### PR TITLE
Initial implementation

### PR DESCRIPTION
**Pull request by taion**

### CODE REVIEW

File: oauth_token_manager.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import time
+
+import requests
+
+
+class OAuthTokenManager(object):
+    def __init__(
+        self,
+        url,
+        *,
```
Comment (itajaja) [inline oauth_token_manager.py:10]: no python2 support? 😱

**Review by itajaja — APPROVED** [review]

File: oauth_token_manager.py (hunk 1 above)
Comment (jquense) [reply oauth_token_manager.py:10]: Fashion forward

Comment (taion) [reply oauth_token_manager.py:10]: do we need it?

Comment (itajaja) [reply oauth_token_manager.py:10]: jk

**Comment by taion** [comment]
we ok to merge?

**Pull request merged**